source/reference/minio-server/settings/object-lambda.rst

```diff
@@ -10,49 +10,193 @@ Object Lambda Function Settings
    :local:
    :depth: 2
 
-This page documents environment variables for configuring MinIO to publish data to an HTTP webhook endpoint and trigger an Object Lambda function.
-See :ref:`developers-object-lambda` for more complete documentation and tutorials on using these environment variables.
+This page documents settings for configuring MinIO to publish data to an HTTP webhook endpoint and trigger an Object Lambda function.
+See :ref:`developers-object-lambda` for more complete documentation and tutorials on using these settings.
 
-You can specify multiple webhook endpoints as Lambda targets by appending a unique identifier ``_FUNCTIONNAME`` for each Object Lambda function.
-For example, the following command sets two distinct Object Lambda webhook endpoints:
+You can establish or modify settings by defining:
 
-.. code-block:: shell
-   :class: copyable
+- an *environment variable* on the host system prior to starting or restarting the MinIO Server.
+  Refer to your operating system's documentation for how to define an environment variable.
+- a *configuration setting* using :mc:`mc admin config set`.
 
-   export MINIO_LAMBDA_WEBHOOK_ENABLE_myfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_myfunction="http://webhook-1.example.net"
-   export MINIO_LAMBDA_WEBHOOK_ENABLE_yourfunction="on"
-   export MINIO_LAMBDA_WEBHOOK_ENDPOINT_yourfunction="http://webhook-2.example.net"
+If you define both an environment variable and the similar configuration setting, MinIO uses the environment variable value.
+
+Some settings have only an environment variable or a configuration setting, but not both.
```

Comment:
Can't use the standard include text here because these settings aren't in MinIO Console like others.